extensions/amp-analytics/0.1/test/test-amp-analytics.js

```diff
@@ -304,6 +304,7 @@ describes.realWin('amp-analytics', {
       expect(preloadSpy.withArgs(
           'http://localhost:9876/dist/iframe-transport-client-lib.js',
           'script')).to.be.calledOnce;
+      delete analytics.predefinedConfig_['foo'];
```

Comment:
Can you use stub for this?  I'm concerned that if the test fails prior to your delete, then the object will never be cleaned.  The other option is to add a value just for testing (see RTC example: https://github.com/ampproject/amphtml/blob/master/extensions/amp-a4a/0.1/callout-vendors.js#L72)

Reply:
Cool, I didn't realize you could stub non-functions.
Done.